src/modules/DataCatalog/components/ViewEdit/Edit/FormCreateDataNode/container.js

```diff
@@ -40,15 +43,35 @@ export default class FormCreateDataNode extends ContainerBuilder {
   mapStateToProps(state) {
     const isLoading = get(state, 'dataCatalog.dataNode.isLoading', false);
     const datanodeId = get(state, 'dataCatalog.dataSource.data.result.dataNode.id');
+
+    const formValues = selectors.getValues(state);
+
+    let fullFilled = false;
+    let existedDataNode = false;
+    if (!!formValues) {
+      if (!datanodeId) {
+        const selectedNodeId = formValues.node;
+        existedDataNode = selectedNodeId !== -1;
+      }
+      if (!existedDataNode) {
+        fullFilled = !!formValues.node && !!formValues.organization && !!formValues.description;
+      }
+    }
     const dataNodes = get(state, 'dataCatalog.dataNode.data', [], 'Array').map((node) => ({
       value: node.centralId,
       label: node.name,
     }));
-
+    const organizations = get(state, 'dataCatalog.organization.data', [], 'Array').map((organization) => ({
+      value: organization.id,
+      label: organization.name,
+    }));
     return {
       isLoading,
       datanodeId,
       dataNodes,
+      organizations,
+      existedDataNode,
+      fullFilled,
```

Comment:
instead of checking this value here it's better to move it to separate method an pass it to getFormParams():
getFormParams() {
     return {
       form: forms.createDataNode,
       validate: this.isFormFilled,
     };
   }